src/main/resources/swagger/cromwell.yaml

```diff
@@ -507,6 +507,75 @@ paths:
       security:
         - google_oauth:
             - openid
+  '/workflows/{version}/{id}/call-caching':
+    post:
+      summary: Alter call cache result reuse settings for all calls within the specified workflow.
+      parameters:
+        - name: version
+          description: API Version
+          required: true
+          type: string
+          in: path
+          default: v1
+        - name: id
+          description: Workflow ID
+          required: true
+          type: string
+          in: path
+        - name: allow
+          description: Whether to allow call cache result reuse for all calls within the specified workflow.
+          required: true
+          type: boolean
+          in: query
+      tags:
+        - Workflows
+      responses:
+        '200':
+          description: Successful Request
+        '400':
+          description: Validation error
+        '500':
+          description: Internal Error
+      security:
+        - google_oauth:
+            - openid
+  '/workflows/{version}/{id}/call-caching/{call}':
+    post:
+      summary: Alter call cache result reuse settings for the specified call within the specified workflow.
+      parameters:
+        - name: version
+          description: API Version
+          required: true
+          type: string
+          in: path
+          default: v1
+        - name: id
+          description: Workflow ID
+          required: true
+          type: string
+          in: path
+        - name: call
+          description: Call FQN
```

Comment:
This `name` and `description` don't match the same parameter in other endpoints, which seem to (mostly) be `callFqn` and `Call fully qualified name`.